CONTRIBUTING.md

```diff
@@ -8,6 +8,7 @@
 * Please be specific on how to reproduce the problem.
 * Do not submit duplicate issues. Search first.
 * Avoid including multiple requests/bugs into a single issue.
+* Stay calm and professional when submitting a bug. Bad attitude will drive developers away.
```

Comment:
I agree with @Mailaender.
You could also add "or commenting on a pull request" to the end of the first sentence.